tss-esapi-sys/build.rs

```diff
@@ -7,18 +7,111 @@ fn main() {
         return;
     }
 
+    #[cfg(feature = "bundled")]
+    let source_path;
```

Comment:
I think you should move all of the compiling of the bundled stuff into separate function in the `tpm2_tss` module called `create_installation`, `create_bundled_installation` or `make_installation` or something like that.